### DISCUSSION TITLE
symfony6 incapable de mettre à jour la base de données avec findAll('$datafields' => $form->formfields)

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#47132] [Symfony Book: The Fast Track] [answered]
public function crediter(Request $request,CompteRepository $repo,ManagerRegistry $doctrine,): Response {

    $select = new Select();
    $form = $this->createForm(SelectType::class, $select);
    $form->handleRequest($request);
    $debiter = $repo->findBy(['tel' => $select->getCompteD()]);
    $crediter = $repo->findBy(['tel' => $select->getCompteC()]);
    if ($form->isSubmitted() && $form->isValid()) {
        $manager=$doctrine->getManager();
        $debiteur = $manager->$repo->findBy(['tel' => $select->getCompteD()]);
        $crediteur = $manager->$repo->findBy(['tel' => $select->getCompteC()]);
        $crediteur->setSolde('4466');
        $debiteur->setSolde('506');
        $manager->flush();
        return $this->redirectToRoute('app_operation', []);
        }
}

**Comment by lyrixx** [comment, marked as answer]
(Bonjour, c'est mieux de communiquer en Anglais sur GitHub)

Hello, you have a type here : 

```diff
- $manager->$repo
+ $manager->repo
```
same next line

I guess you get an error telling you that.
